src/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__image {
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+
+  }
+
+  &__product-title {
+    margin-top: 40px;
+    color: var(--main-accent, #060b35);
```

Comment:
1.  please use scss variables - `$main-accent`
2. move all variables to seperate file